docs/release-log.md

```diff
@@ -1,6 +1,58 @@
 # Release Log
 
-## 09.26.23
+## 2023-10-02
```

Comment:
I had used "Part of" for each change and associated with either the dev server, components, or platform change. Not saying not open to changing that pattern but wanted to call it out.

Reply:
added those back in